pkg/cmd/run.go

```diff
@@ -850,7 +860,7 @@ func getMountPath(targetPath string, dirName string, path string) (string, error
 }
 
 // nolint:errcheck
-func uploadPomFromJar(gav maven.Dependency, path string, platform *v1.IntegrationPlatform, ns string, options spectrum.Options) maven.Dependency {
+func uploadPomFromJar(gav maven.Dependency, path string, platform *v1.IntegrationPlatform, ns string, options spectrum.Options, verbose bool) maven.Dependency {
```

Comment:
Maybe we can change it into method of `runCmdOptions` as the args now looks a bit too long.